### PR TITLE
refactor: standalone app and control flow

### PR DESCRIPTION
**Pull request by sheikalthaf**

### CODE REVIEW

**Comment by sheikalthaf** [comment]
@czeckd this is not an update for library, just converted the demo app to standalone and using control flow

File: src/app/demo-app.component.html [hunk 1]
```diff
@@ -22,25 +22,31 @@ <h2>Set the SVG style</h2>
...
-			<span class="px" *ngIf="!autoheight"><input type="number" min="0" max="1000" [(ngModel)]="h" name="height"></span>
+			@if (!autoheight) {
+				<span class="px"><input type="number" min="0" max="1000" [(ngModel)]="h" name="height"></span>
+			}
 			<span style="margin-left:10px;">&nbsp;</span>
 
 			<label>width:</label>
 			<label>auto <input type="checkbox" [(ngModel)]="autowidth" name="autowidth"></label>
-			<span class="px" *ngIf="!autowidth"><input type="number" min="0" max="1000" [(ngModel)]="w" name="width"></span>
+			@if (!autoheight) {
```
Comment (czeckd) [inline src/app/demo-app.component.html]: This should be `!autowidth`.

**Review by czeckd — CHANGES_REQUESTED** [review]
@sheikalthaf Found a typo, but other than that it looks good. Thanks for the update!

File: src/app/demo-app.component.html (hunk 1 above)
Comment (sheikalthaf) [reply src/app/demo-app.component.html]: fixed it

**Comment by sheikalthaf** [comment]
@czeckd fixed the typo

**Pull request merged**